docs/manual/cluster-admin/how-to-set-up-docker-image-cache.md

```diff
@@ -0,0 +1,104 @@
+
+
+## Set Up Docker Image Cache in Installation
+
+In installation, the only effort you need to perform is change config.yaml in contrib/kubespray/config.yaml. Those setting with "docker_cache" substring are related in "OpenPAI Customized Settings" section. 
+
+* `enable_docker_cache`: true if you want to enable docker-cache service, default is false, which make all following params won't take effect.
+* `docker_cache_storage_backend`: storage backend type selector, "azure" is for [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/), "filesystem" is for linux filesystem.
+* `docker_azure_account_name`: required when storage backend is "azure", should be your azure blob storage accountname.
+* `docker_azure_account_key`: required when storage backend is "azure", should be your azure blob storage base64 encoded account key.
+* `docker_cache_azure_container_name`: required when storage backend is "azure", should be modified if you want to specify container name your docker-cache use, default is "dockerregistry".
+* `docker_cache_fs_mount_path`: required when storage backend is "filesystem", should be modified if you want to specify path your docker-cache use, default is "/var/lib/registry".
+* `docker_cache_remote_url`: pull-through cache remote url, should be modified if you want to specify other docker remote registry rather than dockerhub, default is "https://registry-1.docker.io/".
+* `docker_cache_htpasswd`: htpasswd auth info with base64 encoded, should be use with ssl when docker-cache cache some private registry as an access control method.
+
+### `config.yaml` example with azure
+
+``` yaml
+user: forexample
+password: forexample
+docker_image_tag: v1.5.0
+
+# Optional
+
+#######################################################################
+#                    OpenPAI Customized Settings                      #
+#######################################################################
+# enable_hived_scheduler: true
+enable_docker_cache: true
+docker_cache_storage_backend: "azure"
+docker_cache_azure_account_name: "forexample"
+docker_cache_azure_account_key: "forexample"
+# docker_cache_azure_container_name: "dockerregistry"
+# docker_cache_fs_mount_path: "/var/lib/registry"
+# docker_cache_remote_url: "https://registry-1.docker.io"
+# docker_cache_htpasswd: "" 
+# enable_marketplace: "true"
+
+# ...
```

Comment:
Can we provide several different config examples for 1)  set up registry with AzureBlob 2) set up registry with local storage 3) customized registry ? It will be more clear to me.